lego/apps/ical/viewsets.py

```diff
@@ -89,21 +89,11 @@ def personal(self, request):
         permission_handler = get_permission_handler(Event)
         following_events = permission_handler.filter_queryset(
             request.user,
-            Event.objects.filter(
-                followers__follower_id=request.user.id,
-                end_time__gt=timezone.now()
-                - timedelta(days=constants.HISTORY_BACKWARDS_IN_DAYS),
-            ).all(),
+            Event.objects.filter(followers__follower_id=request.user.id).all(),
```

Comment:
If `constants.HISTORY_BACKWARDS_IN_DAYS` is no longer used (I am not sure tho), it should be removed from the constants file. Just search through the repo after `HISTORY_BACKWARDS_IN_DAYS`.

Reply:
This constant is still used for all events, thus it should remain.